joomla-projects/media-manager-improvement
# Aspect ratio in crop view - Replace radio buttons with drop down list

Pull Request for Issue #502, #499 

@dgt41 
> For the other issue let’s add a first option: original aspect ratio which should be selected by default

### Summary of Changes
- Replaced the radio buttons with a drop down.
- Added a default to use original image aspect ratio
- Added Landscape and Portrait orientation alternatives
- Changed the Reset to read "No aspect ratio" for cropping without restrictions.

### Testing Instructions
Click on the different options in the drop down and use the Ratio aspects to crop the images.

Please check the Javascript. I'm not that fit and the Dom results has changed in 4.0

Image 1: The field, Image 2 Drop down opened. 
![aspect-ratio_1](https://user-images.githubusercontent.com/956537/35180547-1d266e7c-fdb3-11e7-821c-69a08dc18cbf.jpg)
![aspect-ratio_2](https://user-images.githubusercontent.com/956537/35180550-2538987e-fdb3-11e7-821a-f5cf99cef3af.jpg)







### Documentation Changes Required
